components/date-range-picker.tsx

```diff
@@ -101,6 +101,13 @@ export function DateRangePicker({
                   return null
                 }
 
+                const urlDate = (dateString: string) => {
```

Comment:
@redpangilinan I believe with this new function we can remove/replace the previous urlDate function at the top of this file as it's no longer used.

Reply:
That's right. I'll look for more stuff that I overlooked, I'm pretty sure there's one or two unused imports and I'm going to clean everything later. Thanks!